DynamicModeler/Logic/vtkSlicerDynamicModelerPlaneCutTool.cxx

```diff
@@ -177,7 +177,7 @@ const char* vtkSlicerDynamicModelerPlaneCutTool::GetName()
 }
 
 //----------------------------------------------------------------------------
-void vtkSlicerDynamicModelerPlaneCutTool::CreateEndCap(vtkPolyData* polyData, vtkPlaneCollection* planes, vtkPolyData* originalPolyData, vtkImplicitBoolean* cutFunction, vtkPolyData* outputEndCap)
+void vtkSlicerDynamicModelerPlaneCutTool::CreateEndCap(vtkPolyData* vtkNotUsed(polyData), vtkPlaneCollection* planes, vtkPolyData* originalPolyData, vtkImplicitBoolean* cutFunction, vtkPolyData* outputEndCap)
```

Comment:
Thanks @jcfr, sorry I didn't catch this in the refactor. The argument is no longer needed, so it would be fine to just delete it.